platforms/documentation/docs/src/snippets/dependencyManagement/customizingResolution-metadataRule/groovy/build.gradle

```diff
@@ -27,14 +27,14 @@ abstract class TargetJvmVersionRule implements ComponentMetadataRule {
 }
 dependencies {
     components {
-        withModule("commons-io:commons-io", TargetJvmVersionRule) {
+        withModule("commons-io:commons-io:2.15.1", TargetJvmVersionRule) {
             params(7)
         }
         withModule("commons-collections:commons-collections", TargetJvmVersionRule) {
             params(8)
         }
     }
-    implementation("commons-io:commons-io:2.6")
+    implementation("commons-io:commons-io:2.15.1")
```

Comment:
💅 This is a snippet from the docs, it does not need to have the latest deps versions for everything